### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by scjkk-jo**
Skeleton for final README file

I added the list of contributors and a basic project description.

We need:
- a description of the theoretical solution implemented
- presentation of the results
- instructions for using the files

### CODE REVIEW

**Review by mostafa8310 — COMMENTED** [review]
The structure looks good, but not all team members are included. Please also send me the final version when it is finished.

**Review by KasiaNowakowska — COMMENTED** [review]
remove Rhiannon lol

File: README.md [hunk 1]
```diff
@@ -1 +1,34 @@
...
+## Finding the area of a circle given the radius, without using pi
+
+### List of contributors
+ 
+
+
+| Issue No | Project                                                                                                                                | Assignee                                                  |
+| ----- | -------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------- |
+| 1     | Writing the README      | Jo                |
+| 2     | Adding a license        |  Meg |
```
Comment (teleanuc) [inline README.md:12]: Missing initial issue (I believe it was #2) - Kasia worked on creating .gitignore file

**Review by mostafa8310 — COMMENTED** [review]
Please include the module leader names as well. Maybe a good idea to start writing up Theoretical solution. You can get help from Meg.

**Review by seuncoker — COMMENTED** [review]
please, double check that the issues number is right.

Thanks.

**Review by teleanuc — APPROVED** [review]
Looks good to me :)

**Review by mostafa8310 — APPROVED** [review]
Nice job Jo!

**Pull request merged**